imager/imager_site/settings.py

```diff
@@ -31,12 +31,15 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'imager_profile',
+    'imager_images',
```

Comment:
Your own apps should be listed underneath the `django.contrib.*` stuff - otherwise, you end up needing to manually specify the apps in a specific order when you try to migrate.